Documentation/deploy-master-single.md

```diff
@@ -336,7 +336,7 @@ Ensure that the kublet will start after a reboot:
 $ sudo systemctl enable kubelet
 ```
 
-Our Pods should now we starting up and downloading their containers. To check the download progress, you can run `docker ps`.
+Our Pods should now starting up and downloading their containers. To check the download progress, you can run `docker ps`.
```

Comment:
I believe the `we` was intended to be a `be`. Now the tense of `starting` and `downloading` are off. You should choose one of the following:

```Our Pods should now be starting up and downloading their containers.```

```Our Pods should now start up and download their containers.```